### PR TITLE
Issue359

### PR DESCRIPTION
**Pull request by ndegwamartin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,4 @@ build/
 opensrp-path/fabric.properties
 opensrp-path/reports/
 opensrp-app/reports/
+opensrp-app/jacoco.exec
```
Comment (githengi) [inline .gitignore]: Why is this removed. This file will be committed when running tests locally

Comment (ndegwamartin) [reply .gitignore]: It's been ignored to avoid that very case

File: opensrp-app/src/main/java/org/smartregister/util/JsonFormUtils.java [hunk 2]
```diff
@@ -809,20 +809,22 @@ public static void fillSubFormAddressFields(JSONObject jsonObject, Map<String, A
...
     public static JSONArray fields(JSONObject jsonForm) {
         try {
 
-            JSONObject step1 = jsonForm.has(STEP1) ? jsonForm.getJSONObject(STEP1) : null;
-            if (step1 == null) {
-                return null;
-            }
-
-            return step1.has(FIELDS) ? step1.getJSONArray(FIELDS) : null;
+            return getMultiStepFormFields(jsonForm);
```
Comment (githengi) [inline opensrp-app/src/main/java/org/smartregister/util/JsonFormUtils.java]: This method has added complexity than the method being removed if the form has only 1 step. Can `getMultiStepFormFields` be refactored so that if a form has 1 step then ` step1.getJSONArray(FIELDS) ` is used

Comment (ndegwamartin) [reply opensrp-app/src/main/java/org/smartregister/util/JsonFormUtils.java]: Figured that to be a negligible trade off given 1. its duplicating the exact same functionality and 2. in the case that it is only one step in the form being processed, the loop only executes once.
